fl4health/utils/losses.py

```diff
@@ -58,97 +55,183 @@ class LossMeterType(Enum):
     ACCUMULATION = "ACCUMULATION"
 
 
+class LossType(Enum):
+    BACKWARD = "BACKWARD"
+    ADDITIONAL = "ADDITIONAL"
+
+
 class LossMeter(ABC):
     @abstractmethod
     def update(self, losses: Losses) -> None:
+        """
+        Appends loss to list of losses.
+
+        Args:
+            losses (Losses): A losses object with checkpoint, backward and additional losses.
+
+        Raises:
+            NotImplementedError: To be implented by child classes.
+        """
         raise NotImplementedError
 
     @abstractmethod
     def clear(self) -> None:
+        """
+        Reset the meter by re-initializing losses_list to be empty
+
+        Raises:
+            NotImplementedError: To be implemented by child class.
+        """
         raise NotImplementedError
 
     @abstractmethod
     def compute(self) -> Losses:
+        """
+        Compute aggregation of current list of losses if non-empty.
+
+        Returns:
+            Losses: New Losses object with average of losses in losses_list.
```

Comment:
This doesn't necessarily have to be the "average" of the losses. Could just say aggregation again?

Reply:
Yup, good catch!